packages/esm-patient-notes-app/src/notes/visit-notes-form.component.tsx

```diff
@@ -385,7 +385,9 @@ const VisitNotesForm: React.FC<DefaultWorkspaceProps> = ({ closeWorkspace, patie
           </Column>
         </Row>
         <Row className={styles.row}>
-          <Column sm={1}></Column>
+          <Column sm={1}>
+            <span className={styles.columnLabel}>{t('secondaryDiagnosis', 'Secondary Diagnosis')}</span>
```

Comment:
Updated titles in the Diagnosis section to match the [design](https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/60d5e3a96011db1220bd41f4)